src/pages/map/index.tsx

```diff
@@ -63,14 +68,19 @@ const LandmarkTabPanel = ({ latitude }: { latitude: Latitude }) => {
 };
 
 const LandmarkCard = ({ landmark }: { landmark: Landmark })  => {
-  return (
-    <Card>
-      <CardHeader>
-        <Heading size="md">{landmark.名称}</Heading>
+  const maxChar: number = useBreakpointValue({ base: mobileTextLimit, 
+    [sidebarBreakpoint]: desktopTextLimit }) || desktopTextLimit;
+  const cardText = landmark.定义.length <= maxChar? landmark.定义 : <>
+    <span>{landmark.定义.substring(0, maxChar)}...{' '}</span>
```

Comment:
`<Text>{landmark...}<Link>...</Link></Text>` here and remove `<Text>` at line 83.